### PR TITLE
docker-swarm: Remove TLS, offer SSH tunnel for cluster communication

### PR DESCRIPTION
**Pull request by ahmetalpbalkan**
Template now offers SSH tunneling commands as output to connect to the cluster. Removed swarm manager endpoint from the load balancer and the docs.

image

# Example

    sshTunnelCmd: ssh -L 2375:swarm-master-0:2375 -N core@ahmetswarm5-manage.westus.cloudapp.azure.com -p 2200
    dockerCmd: docker -H tcp://localhost:2375 info

@rgardler

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@ahmetalpbalkan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ahmetb). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: docker-swarm-cluster/README.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 
-*(Built by: [ahmetalpbalkan](https://github.com/ahmetalpbalkan), [garimakhulbe](https://github.com/garimakhulbe))*
+*(Built by: [ahmetalpbalkan](https://github.com/ahmetalpbalkan),
```
Comment (singhkay) [inline docker-swarm-cluster/README.md]: Can you remove this "Built by" declaration?

Comment (ahmetalpbalkan) [inline docker-swarm-cluster/README.md]: @singhkay This will remove credit to the second author.

Comment (ahmetalpbalkan) [inline docker-swarm-cluster/README.md]: @singhkay eh fixed, better than stalling.

Comment (singhkay) [inline docker-swarm-cluster/README.md]: @ahmetalpbalkan Thanks! I looked through the git history and it seems the garimakhulbe wrote the other template https://github.com/Azure/azure-quickstart-templates/tree/master/docker-swarm-cluster-simple. Not sure how her name got in this readme.

**Comment by ahmetalpbalkan** [comment]
Because her template was the basis of this template, I gave credit but now removing b/c of the ACOM website requirement.

**Pull request merged**